src/libraries/System.Runtime.Serialization.Formatters/tests/DisableBitTests.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using Microsoft.DotNet.RemoteExecutor;
+using Xunit;
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    public static class DisableBitTests
+    {
+        // these tests only make sense on platforms with both SecureAppContext and RemoteExecutor support
+        public static bool ShouldRunTests => !PlatformDetection.IsNetFramework && RemoteExecutor.IsSupported;
+
+        private const string EnableBinaryFormatterSwitchName = "System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization";
+        private const string MoreInfoUrl = "https://aka.ms/binaryformatter";
+
+        [ConditionalFact(nameof(ShouldRunTests))]
+        public static void DisabledThroughAppContext()
```

Comment:
Should we add a test for `Browser` (RemoteExecutor is not supported there) to validate that we correctly throw not supported when `Serialize/Deserialize` is called?